demisto_sdk/core.py

```diff
@@ -126,3 +129,12 @@ def lint(self, project_dir: str, **kwargs):
         """
         linter = Linter(configuration=self.configuration, project_dir=project_dir, **kwargs)
         return linter.run_dev_packages()
+
+    def secrets(self, **kwargs):
+        sys.path.append(self.configuration.content_dir)
+
+        print_color('Starting secrets detection', LOG_COLORS.GREEN)
+
+        validator = SecretsValidator(configuration=self.configuration, **kwargs)
+
+        return validator.main()
```

Comment:
Do we really want to call it `main`?

Reply:
I will rename it to something cheesy soon